source/lib/login.ts

```diff
@@ -33,11 +33,8 @@ type Credentials = {
 	password: string
 }
 
-async function loadCredentials(): Promise<SharedWebCredentials> {
+async function loadCredentials(): Promise<false | SharedWebCredentials> {
 	let credentials = await getInternetCredentials(SIS_LOGIN_KEY)
-	if (credentials === false) {
-		throw new NoCredentialsError()
-	}
 	return credentials
 }
```

Comment:
I like dropping the throw-point and flattening out the type signature.

`false` still seems like a weird return value for a call like this. Could we do a bit of legwork and remap `false` -> `null`? (Then the return of this is `SharedWebCredentials | null`.)

Reply:
That makes a lot more sense to me.

Reply:
Makes sense!